doc/source/extending.rst

```diff
@@ -113,21 +113,67 @@ by some other storage type, like Python lists.
 See the `extension array source`_ for the interface definition. The docstrings
 and comments contain guidance for properly implementing the interface.
 
+.. _extending.extension.operator:
+
 :class:`~pandas.api.extensions.ExtensionArray` Operator Support
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 By default, there are no operators defined for the class :class:`~pandas.api.extensions.ExtensionArray`.
-There are two ways that you can provide operator support for your ExtensionArray. One
-is to define each of the operators on your ExtensionArray subclass. The second method
-assumes that the underlying elements of the ExtensionArray have the individual operators
-already defined.  An ``ExtensionArray`` subclass consisting of objects that have arithmetic and
-comparison operators defined on the underlying objects can easily support 
-those operators on the ``ExtensionArray``, and therefore the operators 
-on ``Series`` built on those ``ExtensionArray`` classes will work as expected.
+There are two approaches for providing operator support for your ExtensionArray:
+
+1. Define each of the operators on your ExtensionArray subclass. 
+2. Use operators from pandas defined on the ExtensionArray subclass based on already defined
+   operators on the underlying elements.
+    
+For the first approach, you will need to create a mixin class with a single class method,
```

Comment:
I would not explain it like this. A very likely approach for the first case is that you actually define the operators you want to support by adding `__add__`, `__eq__` implementations etc.

Reply:
I think this is the heart of what @jreback was getting at, i.e., would one create each operator independently, or create one `_create_method()` implementation that takes care of all of the operators.